### ISSUE TITLE
Flashing Green

### ISSUE THREAD

**Issue by shme14** [cpebit/aawireless-issues#191]
**Describe the bug**
As of yesterday after a few cycles of use the device will just flash green forever and display a notification that says "Looking for Android Auto" . I have tried both the default 8.8 version of AA and the newer 8.9 version of AA. If I clear the storage of AA it will work again for a random amount of times. (Usually only once). Before yesterday it has worked flawlessly for over a year.

I can plug my phone straight into the stereo and it works instantly every time.

**Device name (AAWireless/AndroidAuto-AAWxxxxxxx) (please complete the following information):**
 - AAWireless/AndroidAuto-AAWFLPjbBrs

**Car (please complete the following information):**
 - Eonon head unit

**Smartphone (please complete the following information):**
 - Device: Pixel 6

**Comment by Snirpo** [comment]
Is the device still paired with the phone?

**Comment by shme14** [comment]
Yes. I updated joined the beta program for the app and disabled battery optimization for the android auto app. It seems to be working fine again. I'm not sure if it's something I did or just a fluke though.

**Comment by Snirpo** [comment]
Hmm ok, is it possible to go back to the normal (non-beta) version of the app and check if it starts happening again? I'm curious if this is somehow caused by the companion app.
